accounting/localizations/argentina.rst

```diff
@@ -0,0 +1,280 @@
+2.3.1 Use on Invoices
+^^^^^^^^^^^^^^^^^^^^
+The document type on each transaction will be determined by:
+- The Journal related to the Invoice, identifying if the journal use documents.
+- Condition applied based on the type of Issues and Receiver (ex. Type of fiscal regimen of the buyer and type of fiscal regimen of the vendor)
+
+4.4 Journals
+++++++++++++
+In the Argentinean localization the Journal can have a different approach depending on its usage and internal type, to configure you journals go to Accounting -> Configuration -> Journals:
+
+
+4.4.1 AFIP Information
+^^^^^^^^^^^^^^^^^^^^
+
+.. image:: media/argentina10.png
+   :align: center
+
+#. **AFIP POS System**: This field is only visible for the Sales  journals and defined the type of AFIP POS that will be used to manage the transactions for which the journal is created.  The AFIP POS defines as well:
+- The sequences of document types related to the WS.
+- The structure and data of the electronic invoice file.
+
+
+#. **AFIP POS Number**: This is the number configured in the AFIP to identify the operations related to this AFIP POS.
+
+#. **AFIP POS Address**: This field is related to commercial address registered for the POS, which is usually the same address than the Company. For example: has multiple stores (fiscal locations) then AFIP will require that you have one AFIP POS per location: this location will be printed in the invoice report.
+
+#. **Unified Book**: When AFIP POS System is Preimpresa the document types (applicable to the journal)  with the same letter will share the same sequence.  For example
+
+Invoice: FA-A 0001-00000002
+Credit Note: NC-A 0001-00000003
+Debit Note: ND-A 0001-00000004
+
+
+Usage and testing
+~~~~~~~~~~~~~~~~~
+
+1. Invoice
+----------
+
+After the partners and journals are created and configured, when the invoices are created the will have the next behaviour:
+
+1.1 Document type assignation
+++++++++++++++++++++
+Once the partner is selected the document type will filled automatically, based on the AFIP document type:
+
+**Invoice for a customer IVA Responsable Inscripto, prefix A**
+
+.. image:: media/argentina11.png
+   :align: center
+
+**Invoice for an end customer, prefix B**
+
+.. image:: media/argentina12.png
+   :align: center
+
+**Exportation Invoice, prefix E**
+
+.. image:: media/argentina13.png
+   :align: center
+
+As it is shown in the invoices, all of them use the same journal but the prefix and sequence is given by the document type.
+
+The most common document type will be defined automatically for the different combinations of AFIP responsibility type but it can be updated manually by the user.
+
+
+1.2 Invoice Taxes
+++++++++++++++++++++++
+Based on the  AFIP Responsibility type,  the VAT tax can have a different behavior on the pdf report:
+
+
+
+#. Tax excluded: In this case the taxed amount needs to be clearly identified in the report. This condition applies when the customer have the next AFIP Responsibility Type.
```

Comment:
for tax excluded the invoice "a" is the one, so I would remove image:: media/argentina14.png from this section